pages/sunos/truss.md

```diff
@@ -2,7 +2,7 @@
 
 > Troubleshooting tool for tracing system calls.
 > SunOS equivalent of strace.
-> More information: <https://www.unix.com/man-page/linux/1/truss>.
+> More information: <https://manned.org/truss>.
```

Comment:
```suggestion
> More information: <https://www.unix.com/man-page/linux/1/truss>.
```

Reply:
Because you changed `svcs.md`, I thought there was some deeper reason to use unix.com.